crates/ruff_python_ast/ast.toml

```diff
@@ -193,23 +203,25 @@ fields = [
   { name = "handlers", type = "ExceptHandler*" },
   { name = "orelse", type = "Stmt*" },
   { name = "finalbody", type = "Stmt*" },
-  { name = "is_star", type = "bool" },
+  { name = "is_star", type = "bool", skip_source_order = true },
 ]
 
 [Stmt.nodes.StmtAssert]
 doc = "See also [Assert](https://docs.python.org/3/library/ast.html#ast.Assert)"
 fields = [{ name = "test", type = "Expr" }, { name = "msg", type = "Expr?" }]
+source_order = ["test", "msg"]
```

Comment:
Whatäs the reason that we need to specify `source_order` here. It seems to be in the same order as `fields`.

Reply:
My mistake. Removed them.